src/ap/hostapd.c

```diff
@@ -167,18 +167,18 @@ int generate_hostapd_conf(struct apconf *hconf, struct radius_conf *rconf) {
 }
 #endif
 
+void get_hostapd_args(const char *hostapd_bin_path,
+                      const char *hostapd_file_path,
+                      const char *hostapd_log_path,
+                      const char *argv[static 5]) {
```

Comment:
In case you're wondering what this syntax is, see https://beej.us/guide/bgc/html/split/arrays-part-ii.html#static-for-arrays-in-parameter-lists

Essentially, you're guaranteeing to the compiler that `argv` will always have at **least** 5 elements. This is usually good from an auto-vectorization/SIMD standpoint.